src/map/instance.c

```diff
@@ -794,6 +794,27 @@ static void instance_force_destroy(struct map_session_data *sd)
 	}
 }
 
+/**
+ * reloads the map flags from the source map
+ *
+ * @param instance_id
+ */
+static void instance_reload_map_flags(int instance_id)
+{
+	Assert_retv(instance->valid(instance_id));
+
+	struct instance_data *curInst = &instance->list[instance_id];
+
+	for (int i = 0; i < instance->list[instance_id].num_map; i++) {
+		struct map_data *dstMap = &map->list[curInst->map[i]];
+		struct map_data *srcMap = &map->list[dstMap->instance_src_map];
```

Comment:
mark this as const as well

Reply:
thanks - I haven't used const in c before.  Interesting it can pick up dstMap shouldn't be const.  I'm used to javascript